src/main/resources/templates/c_plus_plus/test/testUtils/AbstractTest.py

```
@@ -0,0 +1,274 @@
+                except Exception as e:
+                    self.__markAsFailed(f"'{self.name}' had an internal error. {str(e)}.\nPlease report this to an instructor!")
+                    print_exc()
+                    self._onFailed()
+        else:
+            # Run without timeout:
+            try:
+                self._run()
+            except TestFailedError:
+                printTester(f"'{self.name}' failed.")
+            except Exception as e:
+                self.__markAsFailed(f"'{self.name}' had an internal error. {str(e)}.\nPlease report this to an instructor!")
```

Ruff

Use explicit conversion flag

Replace with conversion flag

(RUF010)